.changeset/lucky-models-switch.md

```
@@ -0,0 +1,10 @@
+---
+'@mastra/factory': patch
+'@mastra/core': patch
+'@mastra/code-sdk': patch
+'mastracode': patch
+---
+
+Add a bulk action that applies a Factory project's default model to its already-running work and review sessions. Previously, changing the default model only affected sessions started afterwards, so recovering from a rate-limited model meant switching the model by hand in every open session. `POST /web/factory/projects/:id/apply-default-model` now walks the project's active run bindings and, for each one, writes the project default as the per-mode model on the bound thread. Runs pick it up at their next start, the same path multiplayer model changes already travel. The action is scoped by the work item behind each binding: bindings whose item is gone, belongs to another project, or has left the active stages are skipped. It writes only thread settings and never rebinds a session's thread or its in-memory model selection, both of which are shared per resource and would change what other threads on that session see. The Factory settings UI exposes this beneath the default model picker as a confirmed action, since it reaches into runs that are already in progress.
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)